### PR TITLE
Add Approximate counting in Java

### PR DESCRIPTION
**Pull request by PaddyKe**

### CODE REVIEW

**Comment by Amaras** [comment]
We'll merge this when #913 is merged, @PaddyKe you can probably use the blueprint provided by the PR so we can merge it rapidly

File: contents/approximate_counting/code/java/ApproximateCounting.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    public static void main(String args[]) {
+        System.out.println("testing 1,000, a = 30, 10% error");
+        testApproximateCount(100, 1_000, 30, 0.1);
+
+        System.out.println("testing 12,345, a = 10, 10% error");
+        testApproximateCount(100, 12_345, 10, 0.1);
+    
+        System.out.println("testing 222,222, a = 0.5, 20% error");
+        testApproximateCount(100, 222_222, 0.5, 0.2);
```
Comment (stormofice) [inline contents/approximate_counting/code/java/ApproximateCounting.java:81]:
```suggestion
        System.out.println("[#]\nCounting Tests, 100 trials");
        System.out.println("[#]\ntesting 1,000, a = 30, 10% error");
        testApproximateCount(100, 1_000, 30, 0.1);

        System.out.println("[#]\ntesting 12,345, a = 10, 10% error");
        testApproximateCount(100, 12_345, 10, 0.1);
    
        System.out.println("[#]\ntesting 222,222, a = 0.5, 20% error");
        testApproximateCount(100, 222_222, 0.5, 0.2);
```

**Review by stormofice — COMMENTED** [review]
These changes should make the implementation's output comply with the standardized version.

**Comment by leios** [comment]
#913 is merged and I commited @stormofice 'ssuggestion, so I will merge this

**Pull request merged**